crates/ruff/src/rules/pyupgrade/settings.rs

```diff
@@ -26,8 +26,8 @@ pub struct Options {
     /// `from __future__ import annotations`. Note that this setting is only
     /// applicable when the target Python version is below 3.9 and 3.10
     /// respectively, and enabling it is equivalent to disabling
-    /// `use-pep585-annotation` (`UP006`) and `use-pep604-annotation`
-    /// (`UP007`) entirely.
+    /// `use-pep585-annotation` (`UP006`), `use-pep604-annotation`
+    /// (`UP007`), and `use-pep604-isinstance` (`UP038`) entirely.
     pub keep_runtime_typing: Option<bool>,
```

Comment:
An `isinstance` call is not an annotation, isn’t affected by `from __future__ import annotations`, and isn’t visible to `typing.get_type_hints` (the upstream motivation for this flag: asottile/pyupgrade#387), so I’m not sure the same logic applies.

Reply:
Don't feel strongly! I thought it could feel inconsistent or surprising to users but your argument makes sense.

https://github.com/charliermarsh/ruff/pull/3328